src/sql/platform/query/common/queryModel.ts

```diff
@@ -32,8 +32,9 @@ export interface IQueryPlanInfo {
 }
 
 export interface IQueryEvent {
-	type: queryeditor.QueryEvent;
+	type: queryeditor.QueryEventType;
 	uri: string;
+	queryInfo: QueryInfo;
```

Comment:
is this type going over the extensibility API correctly?

Reply:
Good catch - no it wasn't because of a circular reference in the JSON. So as a temporary fix for now I'm just going to pass along only the necessary parts (messages and selection information). Eventually it would be good to add more to that but we can do that as the need arises. 

---
In reply to: [310683146](https://github.com/microsoft/azuredatastudio/pull/6579#discussion_r310683146) [](ancestors = 310683146)